config/projects/relman.yml

```
@@ -198,7 +198,7 @@ relman:
         - generic-worker:cache:rust-code-analysis-*
       to: repo:github.com/mozilla/rust-code-analysis:*
     - grant: secrets:get:project/relman/rust-code-analysis/deploy
-      to: repo:github.com/mozilla/rust-code-analysis:tag:*
+      to: repo:github.com/mozilla/rust-code-analysis:*
```

If you want this to work on tags _and_ the `master` branch, I think you'll need something like this:

```suggestion
      to:
        - repo:github.com/mozilla/rust-code-analysis:tag:*
        - repo:github.com/mozilla/rust-code-analysis:branch:master
```

I could always try applying this change locally before we merge and if it's possible to run a quick test to see if this is what you need, that'd be great!